### DISCUSSION TITLE
[Feature] Share via Immich

### DISCUSSION THREAD

**Discussion by marcba78** [immich-app/immich#11953] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add a feature so that Immich appears in the device's "Share" menu. So that when you click share, the photo or video is immediately uploaded to the cloud.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2659
